docs/project-generation-options.rst

```diff
@@ -66,7 +66,8 @@ cloud_provider:
 
     1. AWS_
     2. GCP_
-    3. None
+    3. AZURE_
```

Comment:
In restructured text, this is a link, which target should be specified later in the file:

https://github.com/cookiecutter/cookiecutter-django/blame/0a8b15615a51f09083ce5386c4e09c5fe2ed2757/docs/project-generation-options.rst#L148-L149

Reply:
Thanks for catching this, updated!